typescript/sdk/src/types.ts

```diff
@@ -26,25 +26,31 @@ export enum Chains {
   ethereum,
   avalanche,
   polygon,
+  test1,
+  test2,
+  test3,
 }
 export type ChainName = keyof typeof Chains;
-export type ChainMap<Value> = Record<ChainName, Value>;
-export type ChainSubsetMap<Networks extends ChainName, Value> = Record<
+export type CompleteChainMap<Value> = Record<ChainName, Value>;
```

Comment:
nit: Does `CompleteChainMap` need entries for mainnet, testnet, and test networks?

Out of scope for this PR, but `MainnetChainMap`, `TestnetChainMap`, might be more useful types